### ISSUE TITLE
Navigators not working properly after jQuery UI update

### ISSUE THREAD

**Issue by torinfo** [thexerteproject/xerteonlinetoolkits#381]
Aparently the tabNav, tabNavExtra and the accNav arent working anymore since the update to a newer version of jQuery UI

If there are images in the tabs, they are intially hidden (opacity 0) and they are scaled and made visible (in that order) in the jQuery UI event that indicates the tabs have been changed.

What happened was that the names of the events have changed. 

In tabNav and tabNavExtr, this was sort of solved by not hiding the images in the first place, but then they were not scaled anymore.

I changed the event names and all seems solved now. I'll fix it in develop and cherry-pick into v3.02 if we agree it should be fixed in v3 as well.

**Comment by torinfo** [comment]
Should I cherry-pick this into 3.0 as well?

**Comment by ronm123** [comment]
Hi Tom I've just been investigating some existing LO's with lot's of accNavs which were reported as not showing images and seem to have this issue e.g. only the first tab image shows and any images in subsequent tabs don't. I replaced the accNav.html with the one from develop and the issue was fixed!

So yes I would say definitely cherry pick this into 3.0 as well. In fact once you've done that and it's merged into the v3 update download on the community site I'd suggest we even send an announcement advising everyone applies the update.
